extensions/wikia/ArticleVideo/OoyalaConfigController.class.php

```diff
@@ -267,10 +271,7 @@ class OoyalaConfigController extends WikiaController {
 				'fontString' => 'n',
 				'fontStyleClass' => 'oo-icon oo-icon-system-settings',
 			],
-			'dismiss' => [
-				'fontFamilyName' => 'ooyala-slick-type',
-				'fontString' => 'e',
-				'fontStyleClass' => 'oo-icon oo-icon-system-close',
+			'dismiss' => [// svg set in skin()
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)